system/src/system_network_diagnostics.cpp

```diff
@@ -0,0 +1,355 @@
+/**
+ ******************************************************************************
+  Copyright (c) 2013-2015 Particle Industries, Inc.  All rights reserved.
```

Comment:
Let's use the current copyright header for new files.

Reply:
updated